packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -541,5 +541,6 @@ export function injectIntoDevTools(devToolsConfig: DevToolsConfig): boolean {
     setRefreshHandler: __DEV__ ? setRefreshHandler : null,
     // Enables DevTools to append owner stacks to error messages in DEV mode.
     getCurrentFiber: __DEV__ ? getCurrentFiberForDevTools : null,
+    getStackByFiberInDevAndProd: getStackByFiberInDevAndProd,
```

Comment:
I guess I could only inject this is the feature flag is on... It's just so nice though 😅 I'd like to always inject it

Reply:
The current version copies the implementation in the devtools. Same thing with getComponentName. The debug tools for hooks is also injected.

That’s kind of a nice property. Sure it’s dependent on the data structures but so is everything else. I removed getComponentName in prod in this change so if we had injected it we would’ve had to keep it in the prod bundle just for devtools.

Can you import this from shared instead perhaps? Or we can reexport it from react-debug-tools. Is there anything stateful that would break that?

Reply:
I think that it could also work on older react versions too and you don’t have to worry about the feature flag.

Reply:
> Can you import this from shared instead perhaps?

Not unless I want to make React's feature flag system work in DevTools too (which I'd rather not).

> Or we can reexport it from react-debug-tools. Is there anything stateful that would break that?

This might work. I will look into it.

Reply:
I think I can just make a DevTools feature flag fork and that might be enough to do this. Seems doable.